### ISSUE TITLE
Algorithm Archive logo design

### ISSUE THREAD

**Issue by leios** [algorithm-archivists/algorithm-archive#600]
# Feature Request
Logo for the Algorithm Archive

## Description

We need a logo for the Algorithm Archive, so I figured we should (finally) create an issue on this and see what people have to say. Maybe we could put all the images here?

We are looking for a logo that has the best theme. I can get a "real artist" to make it ore professional in the end. That said, I would love if we could somehow put code to the design... But it's not that important.

## For Algorithm Archive Developers

- [ ] This feature can be added to the [Master Overview](https://github.com/algorithm-archivists/algorithm-archive/projects/11) (if it cannot be, explain why in a comment below -- lack of technical expertise, not relevant to the scope of this project, too ambitious)
- [ ] There is a timeline for when this feature can be implemented
- [ ] The feature has been added to the [Master Overview](https://github.com/algorithm-archivists/algorithm-archive/projects/11)
- [ ] The feature has been implemented (Please link the PR)

**Comment by julianschacher** [comment]
Here are variations of my take on the AAA-logo. Some of you are probably already familiar with it, since it got created a few months ago and I shared it on Discord.

![AAA_logo_idea_box_with_books_all](https://user-images.githubusercontent.com/26492670/56856769-cf2ba900-6961-11e9-9672-46360985070e.png)

The logo itself consists of the three As of AAA/Arcane Algorithm Archive. However one A is replaced by stripes (so that you have two As and stripes), which give the logo the appearance of a box of books. I think that quite nicely represents the idea of the Arcane Algorithm *Archive*.

**Comment by Trashtalk217** [comment]
This is my personal take on the logo, as posted on the discord sever and a slightly more worked out one.
[Original](https://i.imgur.com/74yoCf5.png)
[More worked out](https://i.imgur.com/M7uJwqm.png)

**Comment by etisdew** [comment]
#2 would best represent this if I'm not mistaken, is there motive behind the four A variant?

**Comment by NiveditJain** [comment]
Hey, I would like to contribute to it. Any particular specifications for the logo.?

**Comment by leios** [comment]
@NiveditJain Not really. We are just coming up with ideas right now. I would love to have some code to generate the logo or an svg or something.

I can also take the best idea and go to a graphic designer later.
